com.adsbynimbus.nimbus/Runtime/Scripts/Nimbus.Internal/Interceptor/ThirdPartyDemand/Vungle/VungleIOS.cs

```diff
@@ -0,0 +1,49 @@
+using System;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using Newtonsoft.Json;
+using Nimbus.Internal.Utility;
+using OpenRTB.Request;
+using UnityEngine;
+
+[assembly: InternalsVisibleTo("nimbus.test")]
+namespace Nimbus.Internal.Interceptor.ThirdPartyDemand.Vungle {
+	#if UNITY_IOS && NIMBUS_ENABLE_VUNGLE
+	internal class VungleIOS : IInterceptor, IProvider {
+		private readonly string _appID;
+		
+		[DllImport("__Internal")]
+		private static extern void _initializeVungle(string appKey);
+
+		[DllImport("__Internal")]
+		private static extern string _fetchVungleBuyerId();
+
+		public VungleIOS(string appID) {
+			_appID = appID;
+		}
+
+		public void InitializeNativeSDK() {
+			_initializeVungle(_appID);
+		}
+
+		public BidRequest ModifyRequest(BidRequest bidRequest, string data) {
+			if (data.IsNullOrEmpty()) {
+				return bidRequest;
+			}
+			if (bidRequest.User.Ext != null) {
```

Comment:
In the case where ext is null it should be instantiated instead of skipping this step.